nym-vpn-core/crates/nym-vpn-cli/src/main.rs

```diff
@@ -5,6 +5,7 @@ use std::{fs, path::PathBuf};
 
 use clap::Parser;
 use commands::{CliArgs, ImportCredentialTypeEnum};
+use futures::channel::mpsc;
```

Comment:
We could probably default to using [tokio::sync::mpsc](https://docs.rs/tokio/latest/tokio/sync/mpsc/index.html) when possible